app/models/dialog/orchestration_template_service_dialog.rb

```diff
@@ -6,11 +6,29 @@ def self.create_dialog(name, template)
 
     def create_dialog(name, template)
       Dialog.new(:name => name, :buttons => "submit,cancel").tap do |dialog|
-        tab = dialog.dialog_tabs.build(:display => "edit", :label => "Basic Information", :position => 0)
-        add_stack_group(template.deployment_options, tab, 0)
-
-        template.parameter_groups.each_with_index do |parameter_group, index|
-          add_parameter_group(parameter_group, tab, index + 1)
+        if template.respond_to?(:parameter_groups_tabbed)
```

Comment:
We should not test `respond_to?` for backward compatibility. The proper way is to introduce `parameter_groups_tabbed` method in the base template model. Default implementation is to provide a single tab named `Basic Information`, and allow the subclass to override with multiple tabs.